internal/main.go

```diff
@@ -26,6 +26,7 @@ import (
 
 func fail(format string, args ...interface{}) {
 	fmt.Fprintf(os.Stderr, format, args...)
+	fmt.Println()
```

Comment:
this should be added to the fmt string where fail() is called instead.

Reply:
IE:

```golang
fmt.Fprintf(os.Stderr, fmt.Sprintf("%s\n", format), args...)
```

or added to each call of `fail()`'s format.

Reply:
added to each call of fail() I think.

Reply:
will do